xray/audit/jas/jasmanager.go

```diff
@@ -129,26 +137,22 @@ func getIacOrSecretsScanResults(resultsFileName, workingDir string, isSecret boo
 		results = report.Runs[0].Results
 	}
 
-	var iacOrSecretResults []utils.IacOrSecretResult
+	var sourceCodeScanResults []utils.SourceCodeScanResult
 	for _, result := range results {
 		// Describes a request to “suppress” a result (to exclude it from result lists)
 		if len(result.Suppressions) > 0 {
 			continue
 		}
-		text := *result.Message.Text
-		if isSecret {
-			text = hideSecret(*result.Locations[0].PhysicalLocation.Region.Snippet.Text)
+		index := utils.GetOrCreateCodeScanResult(result, workingDir, &sourceCodeScanResults)
+		if scanType == utils.Secrets {
```

Comment:
we already have a "scanType" field on Xray scans, let's select a different name here

Reply:
changed to JasScanType